applications: serial_lte_modem: Remove DFU AT commands. (#11504)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

nRF9160: Serial LTE modem
-------------------------

* Removed:

>   * DFU AT commands #XDFUGET, #XDFUSIZE and #XDFURUN are removed.

Reviewer: ```suggestion
  * DFU AT commands ``#XDFUGET``, ``#XDFUSIZE`` and ``#XDFURUN``.
```

>     Commands are not usable without a custom application in the target (nRF52 series) device.

Reviewer: Is this explaining why the commands were removed, or clarifying that they can now only be used by having a custom application?

Author: This is trying to explain why the commands **were** removed.

>   * DFU AT commands #XDFUGET, #XDFUSIZE and #XDFURUN are removed.
>     Commands are not usable without a custom application in the target (nRF52 series) device.

Reviewer: ```suggestion
  * DFU AT commands ``#XDFUGET``, ``#XDFUSIZE`` and ``#XDFURUN`` because they were not usable without a custom application in the target (nRF52 series) device.
```
I think combining the sentences makes the most sense to make sure it reads as a reason, then.
